src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java

```diff
@@ -1514,6 +1546,193 @@ private void performMoveAway(@Nullable final Entity attacker)
         }
     }
 
+    @Override
+    public void travel(Vec3 p_21280_)
```

Comment:
I'm not sure if this is a good idea 😅  what did you change in here compared to vanilla?

Reply:
shouldnt it be enough to just override onclimbable?